core/src/main/scala-2/caliban/schema/ArgBuilderDerivation.scala

```diff
@@ -54,6 +57,24 @@ trait CommonArgBuilderDerivation {
         }
       case None        => Left(ExecutionError(s"Can't build a trait from input $input"))
     }
+
+  private def makeOneOffBuilder[A](ctx: SealedTrait[ArgBuilder, A]): ArgBuilder[A] =
```

Comment:
Minor: typo, should be OneOf

Reply:
Seen other occurrences in the rest of the code, might have to do a search and replace 😄

Reply:
Oops 😅 Fixed